schema.json

```diff
@@ -71,7 +71,8 @@
             "properties": {
               "network": {
                 "type": "string",
-                "description": "e.g. Facebook or Twitter"
+                "description": "e.g. Facebook or Twitter",
+                "format": "uri"
```

Comment:
shouldn't this be in the `url` property instead of the `network`?

Reply:
Second that.

Reply:
Yes. My mistake.